### ISSUE TITLE
SyntaxError: invalid syntax, when import comtypes

### ISSUE THREAD

**Issue by adigrabcad** [enthought/comtypes#251]
Clean install of Python 3.9.9 then use a virtual environment to install dependencies:
```
python -m venv env
env\Scripts\activate
pip3 install -U setuptools
pip3 install -U wheel
pip3 install comtypes==1.1.10
```
Start Python and try to `import comtypes`:
![image](https://user-images.githubusercontent.com/15086504/149624615-8642f8e8-6dad-4dfc-ae9f-bd8283401d89.png)
It works fine if you skip:
```
pip3 install -U setuptools
pip3 install -U wheel
```

**Comment by vasily-v-ryabov** [comment]
Waiting for comtypes PR enthought/comtypes#259 to be tested before releasing `comtypes==1.1.11`.

**Comment by kdschlosser** [comment]
https://github.com/enthought/comtypes/releases/tag/1.1.11

**Comment by junkmd** [comment]
@vasily-v-ryabov 
I think the problem reported in this issue has been resolved in 1.1.11.
I think this issue should be closed.
